charts/vantage-kubernetes-agent/Chart.yaml

```diff
@@ -2,6 +2,6 @@ apiVersion: v2
 name: vantage-kubernetes-agent
 description: Provisions the Vantage Kubernetes agent.
 type: application
-version: 1.0.31
+version: 1.0.32
 appVersion: "1.0.24"
```

Comment:
I'm assuming there's a new version of the agent coming to support this functionality. If so, will wont this need to be the new appVersion?

Reply:
There isn't, this is a chart only change to support labeling the app resource.